test/test_eos_ideal.cpp

```diff
@@ -54,6 +54,14 @@ SCENARIO("Ideal gas entropy", "[IdealGas][Entropy]") {
         auto entropy = host_eos.EntropyFromDensityTemperature(rho, T);
         INFO("Entropy: " << entropy << "  True entropy: " << entropy_true);
         CHECK(isClose(entropy, entropy_true, 1e-12));
+
+        AND_THEN("The free energy agrees") {
+          const Real sie = host_eos.InternalEnergyFromDensityTemperature(rho, T);
+          const Real P = host_eos.PressureFromDensityTemperature(rho, T);
+          const Real G_true = sie + (P / rho) - T * entropy;
+          const Real G = host_eos.GibbsFreeEnergyFromDensityTemperature(rho, T);
+          CHECK(isClose(G, G_true, 1e-12));
+        }
```

Comment:
Maybe the same test with DensityInternalEnergy? Then you would have seen the error above ;).

Reply:
Yep good idea. I added both now.